homeassistant/components/srp_energy/sensor.py

```diff
@@ -14,16 +14,18 @@
 from homeassistant.const import ATTR_ATTRIBUTION, ENERGY_KILO_WATT_HOUR
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
 from .const import (
     ATTRIBUTION,
+    CONF_IS_TOU,
     DEFAULT_NAME,
+    DOMAIN,
     ICON,
     MIN_TIME_BETWEEN_UPDATES,
     SENSOR_NAME,
     SENSOR_TYPE,
-    SRP_ENERGY_DOMAIN,
 )
 
 _LOGGER = logging.getLogger(__name__)
```

Comment:
Depending on how broadly you log across this integration, you could define a single logger in `const.py`:

```python
LOGGER = logging.getLogger(__package__)
```

...and import it as needed.